Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/jee/jsf/UserAuthorizationListener.java

```diff
@@ -141,9 +139,9 @@ public PhaseId getPhaseId() {
 
     private String formatPageName(String pageName) {
         if (pageName.startsWith("/")) {
-            return pageName.substring(1, pageName.length()).toLowerCase();
+            return pageName.substring(1, pageName.length()).toLowerCase(Locale.ENGLISH);
```

Comment:
Don't use toLowerCase(Locale.ENGLISH) here. Instead return the string and use equalsIgnoreCase comparison  in method checkConfiguredDisplay.